pyresample/geometry.py

```diff
@@ -1591,15 +1611,17 @@ def _get_geo_boundary_sides(self, frequency=None):
                      ]
         return lon_sides, lat_sides
 
-    def boundary(self, frequency=None, force_clockwise=False):
+    def boundary(self, vertices_per_side=None, force_clockwise=False, frequency=None):
```

Comment:
@ghiggi @mraspaud what do you think about doing a `*, ` on this to force these parameters to be specified by name? This would future-proof changes so that people who may have been doing `.boundary(True, 5)` get an error that these parameters need to be specified by keyword argument.

Reply:
Sounds good, fixed in bea2050